_sass/_base.scss

```
@@ -318,13 +318,29 @@ footer.sticky-bottom {
   border-bottom: 1px solid var(--global-divider-color);
   text-align: center;
   padding-top: 2rem;
-  padding-bottom: 5rem;
+  padding-bottom: 3rem;
   h1 {
     color: var(--global-theme-color);
     font-size: 5rem;
   }
 }
 
+.tag-list {
+  border-bottom: 1px solid var(--global-divider-color);
+  text-align: center;
```

let's add a bit more white space
```suggestion
  text-align: center;
  padding-top: 1rem;
```